modules/ROOT/pages/dw-operators.adoc

```diff
@@ -749,3 +749,44 @@ payload update {
   }
 }
 ----
+
+[[metadata-assignment-operator]]
+== Metadata Assignment Operator
+
+DataWeave supports the metadata assignment operator `<~`, which enables you to set the metadata of any value without the need of using the `as` operator.
+This is useful for cases where you want to set the metadata and keep the current value type. If coercing to a new type is also required then xref:dataweave-types-coercion.adoc[as] needs to be used.
+
+_Introduced in DataWeave 2.5.0. Supported by Mule 4.5 and later._
+
+The operator syntax is as follows:
+
+```
+<valueB> <~ <objectA>
+```
+
+and it reads `assigning objectA as metadata to valueB`. The arrow points to the value that has the metadata assigned.
+
+
+Versions prior to Mule 4.5 require the following syntax for instance to add the `class` metadata field to an object:
+
+[source,weave,linenums]
+----
+%dw 2.0
+---
+{name: "Ken"} as Object {class: "Person"}
+----
+
+In Mule 4.5, the metadata assignment operator `<~` simplifies the syntax:
+
+[source,weave,linenums]
+----
+%dw 2.0
+---
+{name: "Ken"} <~ {class: "Person"}
+----
+
+=== Metadata
+
+Metadata is defined as any additional information attached to a value that doesn't affect its identity. This means that given any two values
+`A` and `B` which are equal then if we attach metadata to `A` and we call it `A'` then `A == A' == B`. So we can define that a value is formed by the value itself
```

Comment:
```suggestion
`A` and `B` that are equal, if you attach metadata to `A` and call it `A'`, then `A == A' == B`. So you can define that a value that is formed by the value
```

Reply:
we avoid using "we" in our docs per our doc style

we (haha) speak to the user as "you"